pkg/log/status.go

```diff
@@ -110,16 +110,17 @@ func IsTerminal(w io.Writer) bool {
 
 // Start starts a new phase of the status, if attached to a terminal
 // there will be a loading spinner with this status
-func (s *Status) Start(status string, debug bool) {
+func (s *Status) Start(status string, debug bool, json bool) {
 	s.End(true)
+
 	// set new status
 	isTerm := IsTerminal(s.writer)
 	s.status = status
 
 	// If we are in debug mode, don't spin!
 	if !isTerm || debug {
 		fmt.Fprintf(s.writer, prefixSpacing+getSpacingString()+suffixSpacing+"%s  ...\n", s.status)
-	} else {
+	} else if !json {
```

Comment:
why not use IsJSON() function here too?

Reply:
You're right, I should of changed the old code here.. Thanks!